spec/services/proofing/socure/id_plus/input_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Proofing::Socure::IdPlus::Input do
+  let(:user) { build(:user) }
+
+  let(:state_id) do
+    Idp::Constants::MOCK_IDV_APPLICANT_WITH_PHONE
+  end
+
+  subject do
+    described_class.new(
+      **state_id.to_h.slice(*described_class.members),
+      email: user.email,
+    )
+  end
+
+  it 'creates an appropriate instance' do
+    expect(subject.to_h).to eql(
+      {
+        address1: '1 FAKE RD',
+        address2: nil,
+        city: 'GREAT FALLS',
+        state: 'MT',
+        zipcode: '59010-1234',
+
+        first_name: 'FAKEY',
+        last_name: 'MCFAKERSON',
+        middle_name: nil,
+
+        dob: '1938-10-06',
+
+        phone: '12025551212',
+        ssn: '900-66-1234',
+        email: user.email,
```

Comment:
users have multiple emails, should we specify either the first created or most recently used?

Reply:
Good point, probably most recently used? Let me look into how we handle this for other services

Reply:
This is how we do it for ThreatMetrix:

https://github.com/18F/identity-idp/blob/a8977893f15df55fafc14a924f231e3794012bf1/app/jobs/resolution_proofing_job.rb#L92

That eventually finds its way into `Ddp::Proofer#proof`.

Reply:
Ok, so the "which email to use" problem is probably not the proofer's--it will be provided as input